erigon-lib/metrics/set.go

```diff
@@ -46,11 +56,18 @@ func (s *Set) Describe(ch chan<- *prometheus.Desc) {
 	if !sort.SliceIsSorted(s.a, lessFunc) {
 		sort.Slice(s.a, lessFunc)
 	}
+	if !sort.SliceIsSorted(s.av, lessFunc) {
```

Comment:
@bretep try remove this check and always call `sort.Slice` - it may improve speed - if slices are usually non-sorted. (not in this PR).